### PR TITLE
Fix #2: Add a method to --check-not-revoked

### LINKED ISSUE #2: Add CLI check-options such as --check-valid and --check-not-revoked
Add options suitable for scripting environments to ensure that, as examples:

- All PEM arguments are valid (not expired, not too new, are enrolled, not revoked)
- All PEM arguments are not revoked (much looser)

### PR DESCRIPTION
**Pull request by jcjones**
important: This PR depends on #7.

### CODE REVIEW

File: crlite_query/query_cli.py [hunk 1]
```diff
@@ -176,18 +176,27 @@ def main():
...
 
-    for (name, iterator) in to_test:
-        query.print_query(name=name, iterator=iterator)
+    failures = list()
+
+    for (name, generator) in to_test:
+        for result in query.query(name=name, generator=generator):
+            result.print_query_result()
+
+    if failures:
```
Comment (kjacobs-moz) [inline crlite_query/query_cli.py]: Were we supposed to append something to `failures`?

Comment (jcjones) [reply crlite_query/query_cli.py]: Ooo, thank you, something in the rebase killed those lines.

Comment (jcjones) [reply crlite_query/query_cli.py]: https://github.com/mozilla/moz_crlite_query/commit/4b701c2 had it

Comment (jcjones) [reply crlite_query/query_cli.py]: Huh, it's there, see https://github.com/mozilla/moz_crlite_query/pull/8/files#diff-48f6a13a01f0caf2caba6154cb3759c4R200-R201

Comment (kjacobs-moz) [reply crlite_query/query_cli.py]: Ah, my bad!

**Pull request merged**